lib/fw.c

```diff
@@ -373,17 +373,32 @@ void switchtec_fw_perror(const char *s, int ret)
 	fprintf(stderr, "%s: %s\n", s, msg);
 }
 
-struct fw_image_header {
```

Comment:
We can keep use the name of this data structure and only change the field definition.